### PR TITLE
add session note

### PR DESCRIPTION
**Pull request by e-rmkt**
- add todays notes

### CODE REVIEW

File: shell-and-git/26-07-2023.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Heute haben wir über Pull Requests und GitHub was gelernt.
+
+Heute nachmittag beschäftigen wir uns mit HTML!
+
+Schmetttherlyng.
```
Comment (Sarottileon) [inline shell-and-git/26-07-2023.md:5]: typo Schmetterling

Comment (PZhu888) [inline shell-and-git/26-07-2023.md:5]: typo...

**Review by Sarottileon — CHANGES_REQUESTED** [review]

**Review by PZhu888 — CHANGES_REQUESTED** [review]
please change the typo...

File: shell-and-git/26-07-2023.md (hunk 1 above)
Comment (Sarottileon) [inline shell-and-git/26-07-2023.md:5]: typo Schmetterling

**Review by Sarottileon — CHANGES_REQUESTED** [review]

File: shell-and-git/26-07-2023.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+Heute haben wir über Pull Requests und GitHub was gelernt.
+
+Heute nachmittag beschäftigen wir uns mit HTML!
```
Comment (Sarottileon) [inline shell-and-git/26-07-2023.md:3]: typo N

**Review by Sarottileon — CHANGES_REQUESTED** [review]

**Pull request merged**